packages/tinacms/src/auth/TinaCloudProvider.tsx

```diff
@@ -189,6 +198,9 @@ export const TinaCloudProvider = (
   }
 
   setupMedia()
+  if (cms.flags.get('tina-admin') === true) {
+    setupAdmin()
+  }
```

Comment:
I realize we've been doing this with `setupMedia,` but should this be inside `useMemo`?